packages/webapp/src/components/Task/AddSoilAmendmentProducts/ProductCard/ProductDetails.tsx

```diff
@@ -366,6 +366,12 @@ const ProductDetails = ({
             hookFormRegister={register(SUPPLIER, {
               required: interested,
               maxLength: hookFormMaxCharsValidation(255),
+              validate: (value) => {
+                if (value?.trim().length === 0) {
+                  return t('COMMON_ERRORS.UNIT.REQUIRED');
```

Comment:
So it is actually only conditionally required if certifications `interested` is `true`.

So currently this is also validating in the `optional` case.

Reply:
@Duncan-Brain Fixed, thanks for the feedback